src/components/__tests__/Button.test.js

```diff
@@ -40,14 +40,6 @@ describe('Button', () => {
     expect(button).toHaveClass('btn-outline-primary');
   });
 
-  test('applies outline styles with "outline" prop', () => {
-    const {
-      container: {firstChild: button}
-    } = render(<Button color="primary" outline />);
-
-    expect(button).toHaveClass('btn-outline-primary');
-  });
-
```

Comment:
Could we leave this in?

Reply:
For sure! I definitely accidentally deleted it.